go.work.sum

```diff
@@ -785,6 +785,7 @@ github.com/googleapis/gnostic v0.5.5 h1:9fHAtK0uDfpveeqqo1hkEZJcFvYXAiCN3UutL8F9
 github.com/gorilla/handlers v1.4.2 h1:0QniY0USkHQ1RGCLfKxeNHK9bkDHGRYGNDFBCS+YARg=
 github.com/gorilla/mux v1.8.0 h1:i40aqfkR1h2SlN9hojwV5ZA91wcXFOvkdNIeFDP5koI=
 github.com/goware/emailproviders v0.0.0-20200728124719-451ef785cf29 h1:/jHkBu2jbrW6+pxmAMt4tR8W6l/Vk02mnGL/ePrg2+I=
+github.com/goware/emailproviders v0.0.0-20200728124719-451ef785cf29/go.mod h1:2FoXdMtWJEKk3zBo2S5eaduD5Bh/5PIYwSlV66E1XMs=
```

Comment:
`git reset` this file and run `git submodule update --recursive` to reset the `opentelemetry-sdk-workers` submodule.